src/index.js

```diff
@@ -10,13 +10,23 @@ import ActivityRepository from './ActivityRepository';
 import HydrationRepository from './HydrationRepository';
 import SleepRepository from './SleepRepository';
 
-// import userData from './data/users';
 var userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
-                    .then(response => response.json())
-                    .then(data => data.userData);
-import sleepData from './data/sleep';
-import hydrationData from './data/hydration';
-import activityData from './data/activity';
+  .then(response => response.json())
+  .then(data => data.userData)
+  .catch(err => console.log(err));
+var sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData')
+  .then(response => response.json())
+  .then(data => data.sleepData)
+  .catch(err => console.log(err));
+var hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData')
+  .then(response => response.json())
+  .then(data => data.hydrationData)
+  .catch(err => console.log(err));
+var activityData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData')
```

Comment:
There may be a way to DRY this up with parameters but we should get the thing to work first and refactor later.  Nice work!